pykafka/producer.py

```diff
@@ -300,7 +303,7 @@ def stop_owned_brokers():
                 for queue_reader in queue_readers:
                     queue_reader.join()
 
-    def produce(self, message, partition_key=None):
+    def produce(self, message, partition_key=None, timestamp=None):
```

Comment:
Sorry, I'd like to push for this: it's the reason I'm bothering with the whole thing at all. I want to migrate legacy data (including history) to kafka, so I need to be able to recreate the kafka queue as it would have been created at a certain time in the past.